src/main.js

```diff
@@ -95,22 +93,34 @@ function saveMyBook(event) {
   covers.push(formCover.value)
   titles.push(formTitle.value)
   descriptors.push(formDescriptor1.value, formDescriptor2.value)
-
-  //randomCover.setAttribute("src", coverValue);
   currentCover = new Cover (formCover.value, formTitle.value, formDescriptor1.value, formDescriptor2.value);
   viewHome()
 }
 
 function saveCover(){
   if (savedCovers.includes(currentCover) === false) {
     savedCovers.push(currentCover)
-    savedCoversSection.innerHTML += `
-      <section class="mini-cover">
-        <img class="cover-image" src="${currentCover.cover}">
-        <h2 class="cover-title">${currentCover.title}</h2>
-        <h3 class="tagline">A tale of <span class="tagline-1">${currentCover.tagline1}</span> and <span class="tagline-2">${currentCover.tagline2}</span></h3>
-        <img class="price-tag" src="./assets/price.png">
-        <img class="overlay" src="./assets/overlay.png">
-      </section>`
   }
+  displayMiniCovers()
+}
+
+function displayMiniCovers() {
```

Comment:
Good call separate this from the saveCover(). I think it will be necessary when we are deleting covers from the view saved page